jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersViewImpl.java

```diff
@@ -151,15 +151,22 @@ public HTMLElement getElement() {
     public void addTextFilter(final String label,
                               final String placeholder,
                               final Consumer<ActiveFilterItem<String>> callback) {
+        addTextFilter(label, placeholder, false, callback);
+    }
 
+    @Override
+    public void addTextFilter(String label, String placeholder, boolean disableFiltersInputHelp, Consumer<ActiveFilterItem<String>> callback) {
+        filtersInputHelp.setDisabled(disableFiltersInputHelp);
+        jQueryPopover.wrap(filtersInputHelp).destroy();
```

Comment:
@bxf12315 this destroy() method have to be executed only when the disableFiltersInput is true. Currently this code makes the popover is not displayed anymore in any of textFilters

Reply:
yes, you are right. I will check it.